### PR TITLE
Change Bounty Details

### LINKED ISSUE #1928: UX Add Project Type + Approval Settings to Issue Details
### User Story

[comment]: # (As a <user type>, I want to <task> so that <goal>.)
As a funder, I'd like the ability to toggle easily between 'Approval Only' / 'Permissionless' bounties and 'Contest' / 'Cooperative' / 'Traditional' bounties so that I am less prone to unrecoverable user error. 

### Why Is this Needed

[comment]: # (Describe the problem and why this task is needed. Provide description of the current state, what you would like to happen, and what actually happen)
*Summary*: Without this feature, I don't have a good way to make changes to the bounty details if I want to make it approval only / cooperative / etc. once I post the issue. I can only cancel and re-do, which isn't elegant. 

### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: Feature

### Solution
[comment]: # (Provide a summary of the solution and a task list on what needs to be fixed.)
*Summary*: Simple solution would be an 'Edit Bounty Details' option which brings up a modal for changing Project Type / Permissions. 

### Definition of Done
[comment]: # (Any other information that would be useful, bullets are helpful.)

- [ ] Allow an ability for the funder to edit the project type and permissions: 'Approval Only' / 'Permissionless' bounties and 'Contest' / 'Cooperative' / 'Traditional' bounties on the issue details page.
- [ ] Design for the experience when contributor(s) have not started on an issue.
- [ ] Design for the experience when a contributor **has** started on an issue.
- [ ] Update email messaging for the appropriate parties once project types or permissions have been made.
- [ ] Consider the experience of how this issue refreshes in the issue explorer.
- [ ] Make recommendations whether this update should re-sync to the blockchain.
- [ ] Design should take into account responsiveness of the site.
- [ ] Review approach with Gitcoin core team

### Additional Information
Open to interpretation on the best solution.

[comment]: # (Any other information that would be useful, content, screenshots, etc.)
screen shot 2018-08-02 at 4 59 06 pm

### PR DESCRIPTION
**Pull request by pinkiebell**
bounty details ('experience_level', 'project_length', 'bounty_type',
'permission_type', 'project_type')

can be changed if the bounty is still on 'the happy path' by the funder
or staff.

Fixes #1928

### CODE REVIEW

**Comment by pinkiebell** [comment]
Any feedback on this one? 😄

**Comment by owocki** [comment]
it still says WIP in the PR title and theres a few TODOs in the diff.  is this ready for review?

**Comment by pinkiebell** [comment]
Yes, it's WIP because of the comments in the PR. I asked for first review in the linked issue ;)

**Comment by owocki** [comment]
QA notes

1. [doesnt seem like the user can edit any of the details around project type](http://bits.owocki.com/2Q3b3i3j1d0k/Screen%20Shot%202018-08-20%20at%205.45.49%20PM.png)
2. why is the URL for 'change details' so long?  [example URL](http://localhost:8000/bounty/change#0x7b2275726c223a22687474703a2f2f6c6f63616c686f73743a383030302f69737375652f676974636f696e636f2f7765622f313636332f393735222c22637265617465645f6f6e223a22323031382d30382d31375432313a33393a35382e3437323736345a222c226d6f6469666965645f6f6e223a22323031382d30382d32305430363a34353a35382e3436393532385a222c227469746c65223a224e65772061766174617220617373657473222c22776562335f63726561746564223a22323031382d30382d31375432313a33333a33315a222c2276616c75655f696e5f746f6b656e223a223330303030303030303030303030303030303030302e3030222c22746f6b656e5f6e616d65223a22444149222c22746f6b656e5f61646472657373223a22307838396432346136623463636231623666616132363235666535363262646439613233323630333539222c22626f756e74795f74797065223a22222c2270726f6a6563745f6c656e677468223a22556e6b6e6f776e222c22657870657269656e63655f6c6576656c223a22222c226769746875625f75726c223a2268747470733a2f2f6769746875622e636f6d2f676974636f696e636f2f7765622f6973737565732f31363633222c226769746875625f636f6d6d656e7473223a312c22626f756e74795f6f776e65725f61646472657373223a22307830306465346231333135333637336263616532363136623637626638323235303064333235666333222c22626f756e74795f6f776e65725f656d61696c223a226b736f776f636b6940676d61696c2e636f6d222c22626f756e74795f6f776e65725f6769746875625f757365726e616d65223a226f776f636b69222c22626f756e74795f6f776e65725f6e616d65223a224b6576696e204f776f636b69222c2266756c66696c6c6d656e7473223a5b5d2c22696e7465726573746564223a5b7b2270726f66696c65223a7b226964223a343735352c2268616e646c65223a22777269746570726f766964656e6365222c226769746875625f75726c223a2268747470733a2f2f6769746875622e636f6d2f777269746570726f766964656e6365222c226176617461725f75726c223a22687474703a2f2f6c6f63616c686f73743a383030302f7374617469632f6176617461722f777269746570726f766964656e6365222c2275726c223a222f70726f66696c652f777269746570726f766964656e6365227d2c2263726561746564223a22323031382d30382d31385430303a34393a35362e3135383434335a222c2270656e64696e67223a66616c73657d2c7b2270726f66696c65223a7b226964223a313338372c2268616e646c65223a2276696b61736b7961646176222c226769746875625f75726c223a2268747470733a2f2f6769746875622e636f6d2f76696b61736b7961646176222c226176617461725f75726c223a22687474703a2f2f6c6f63616c686f73743a383030302f7374617469632f6176617461722f76696b61736b7961646176222c2275726c223a222f70726f66696c652f76696b61736b7961646176227d2c2263726561746564223a22323031382d30382d31385431383a35353a35342e3635363131335a222c2270656e64696e67223a66616c73657d2c7b2270726f66696c65223a7b226964223a363536362c2268616e646c65223a224a6f7365706844756e6976616e222c226769746875625f75726c223a2268747470733a2f2f6769746875622e636f6d2f4a6f7365706844756e6976616e222c226176617461725f75726c223a22687474703a2f2f6c6f63616c686f73743a383030302f7374617469632f6176617461722f4a6f7365706844756e6976616e222c2275726c223a222f70726f66696c652f4a6f7365706844756e6976616e227d2c2263726561746564223a22323031382d30382d31395430383a33383a33372e3838303336335a222c2270656e64696e67223a66616c73657d5d2c2269735f6f70656e223a747275652c22657870697265735f64617465223a22323031392d30382d31375432313a33333a33315a222c2261637469766974696573223a5b7b2261637469766974795f74797065223a226e65775f626f756e7479222c2263726561746564223a22323031382d30382d31375432313a33393a35392e3234363036345a222c2270726f66696c65223a7b226964223a31392c2268616e646c65223a226f776f636b69222c226769746875625f75726c223a2268747470733a2f2f6769746875622e636f6d2f6f776f636b69222c226176617461725f75726c223a22687474703a2f2f6c6f63616c686f73743a383030302f7374617469632f6176617461722f6f776f636b69222c2275726c223a222f70726f66696c652f6f776f636b69227d2c226d65746164617461223a7b226e65775f626f756e7479223a7b226964223a343932322c227469746c65223a224e65772061766174617220617373657473222c22746f6b656e5f6e616d65223a22444149222c2276616c75655f696e5f657468223a224e6f6e65222c2276616c75655f696e5f746f6b656e223a22333030303030303030303030303030303030303030222c2276616c75655f696e5f757364745f6e6f77223a223330302e30222c22746f6b656e5f76616c75655f696e5f75736474223a2231222c22746f6b656e5f76616c75655f74696d655f706567223a22323031382d30382d31372032313a33393a35382e3637343639392b30303a3030227d2c226f6c645f626f756e7479223a6e756c6c2c2266756c66696c6c6d656e74223a6e756c6c7d2c22626f756e7479223a343932322c22746970223a6e756c6c7d2c7b2261637469766974795f74797065223a2273746172745f776f726b222c2263726561746564223a22323031382d30382d31385430303a34393a35362e3134373431335a222c2270726f66696c65223a7b226964223a343735352c2268616e646c65223a22777269746570726f766964656e6365222c226769746875625f75726c223a2268747470733a2f2f6769746875622e636f6d2f777269746570726f766964656e6365222c226176617461725f75726c223a22687474703a2f2f6c6f63616c686f73743a383030302f7374617469632f6176617461722f777269746570726f766964656e6365222c2275726c223a222f70726f66696c652f777269746570726f766964656e6365227d2c226d65746164617461223a7b226964223a343932322c227469746c65223a224e65772061766174617220617373657473222c22746f6b656e5f6e616d65223a22444149222c2276616c75655f696e5f657468223a224e6f6e65222c2276616c75655f696e5f746f6b656e223a223330303030303030303030303030303030303030302e3030222c2276616c75655f696e5f757364745f6e6f77223a223330302e3030222c22746f6b656e5f76616c75655f696e5f75736474223a22312e3030222c22746f6b656e5f76616c75655f74696d655f706567223a22323031382d30382d31382030303a34373a31352e3731313231322b30303a3030227d2c22626f756e7479223a343932322c22746970223a6e756c6c7d2c7b2261637469766974795f74797065223a2273746172745f776f726b222c2263726561746564223a22323031382d30382d31385431383a35353a35342e3635303630305a222c2270726f66696c65223a7b226964223a313338372c2268616e646c65223a2276696b61736b7961646176222c226769746875625f75726c223a2268747470733a2f2f6769746875622e636f6d2f76696b61736b7961646176222c226176617461725f75726c223a22687474703a2f2f6c6f63616c686f73743a383030302f7374617469632f6176617461722f76696b61736b7961646176222c2275726c223a222f70726f66696c652f76696b61736b7961646176227d2c226d65746164617461223a7b226964223a343932322c227469746c65223a224e65772061766174617220617373657473222c22746f6b656e5f6e616d65223a22444149222c2276616c75655f696e5f657468223a224e6f6e65222c2276616c75655f696e5f746f6b656e223a223330303030303030303030303030303030303030302e3030222c2276616c75655f696e5f757364745f6e6f77223a223330302e3030222c22746f6b656e5f76616c75655f696e5f75736474223a22312e3030222c22746f6b656e5f76616c75655f74696d655f706567223a22323031382d30382d31382031383a34363a35382e3939353033322b30303a3030227d2c22626f756e7479223a343932322c22746970223a6e756c6c7d2c7b2261637469766974795f74797065223a2273746172745f776f726b222c2263726561746564223a22323031382d30382d31395430383a33383a33372e3836363934325a222c2270726f66696c65223a7b226964223a363536362c2268616e646c65223a224a6f7365706844756e6976616e222c226769746875625f75726c223a2268747470733a2f2f6769746875622e636f6d2f4a6f7365706844756e6976616e222c226176617461725f75726c223a22687474703a2f2f6c6f63616c686f73743a383030302f7374617469632f6176617461722f4a6f7365706844756e6976616e222c2275726c223a222f70726f66696c652f4a6f7365706844756e6976616e227d2c226d65746164617461223a7b226964223a343932322c227469746c65223a224e65772061766174617220617373657473222c22746f6b656e5f6e616d65223a22444149222c2276616c75655f696e5f657468223a224e6f6e65222c2276616c75655f696e5f746f6b656e223a223330303030303030303030303030303030303030302e3030222c2276616c75655f696e5f757364745f6e6f77223a223330302e3030222c22746f6b656e5f76616c75655f696e5f75736474223a22312e3030222c22746f6b656e5f76616c75655f74696d655f706567223a22323031382d30382d31392030373a34363a34312e3931333636362b30303a3030227d2c22626f756e7479223a343932322c22746970223a6e756c6c7d5d2c226b6579776f726473223a224a6176615363726970742c20507974686f6e2c2048544d4c2c204353532c205368656c6c2c204d616b6566696c652c20446f636b657266696c65222c2263757272656e745f626f756e7479223a747275652c2276616c75655f696e5f657468223a6e756c6c2c22746f6b656e5f76616c75655f696e5f75736474223a22312e3030222c2276616c75655f696e5f757364745f6e6f77223a223330302e3030222c2276616c75655f696e5f75736474223a223330302e3030222c22737461747573223a226f70656e222c226e6f77223a22323031382d30382d32305432333a34343a34362e3636373433395a222c226176617461725f75726c223a22687474703a2f2f6c6f63616c686f73743a383030302f7374617469632f6176617461722f676974636f696e636f222c2276616c75655f74727565223a223330302e3030222c2269737375655f6465736372697074696f6e223a2223232320557365722053746f72795c725c6e5c725c6e4173206120757365722c20492077616e7420746f206e6577206176617461722061737365747320746f2073686f77206f666620616e642065787072657373206d7973656c66206469676974616c6c792e5c725c6e5c725c6e232323204465736372697074696f6e5c725c6e5c725c6e2a547970652a3a20204e657720696d706c656d656e746174696f6e5c725c6e5c725c6e23232320446566696e6974696f6e206f6620446f6e655c725c6e5c725c6e2d205b205d2043726561746520616e202e73766720617373657420616e6420706c61636520697420696e20606170702f6173736574732f76322f696d616765732f6176617461722f3c63617465676f72793e605c725c6e2d205b205d204164642074686520535647206173736574206173206120602e7478746020446a616e676f2074656d706c61746520696e20606170702f6176617461722f74656d706c617465732f3c63617465676f72793e2f3c6974656d7c696e6465783e2e747874602077697468207661726961626c652066696c6c2e5c725c6e2d205b205d2050726f7669646520612073637265656e73686f74206f6620746865206e65772061737365742066726f6d2074686520617661746172206275696c64657220414e44207468652070726f66696c652064657461696c732070616765206f6e2074686973204769746875622069737375652e5c725c6e5c725c6e232323204164646974696f6e616c20496e666f726d6174696f6e5c725c6e5c725c6e4665656c206672656520746f207375626d697420616e792061737365747320796f7520776f756c64206c696b6520746f20736565206f6e20476974636f696e2e20204966207468652061737365742069732061636365707465642c20796f75722061737365742077696c6c206265636f6d6520612070617274206f6620746865206c69766520617661746172206275696c646572212020416c6c206f74686572206173736574732077696c6c20626520657363616c6174656420746f2074686520636f7265207465616d20666f722072657669657720616e6420706f74656e7469616c206d6f64696669636174696f6e2e5c725c6e5c725c6e53656520746865205b617661746172206974656d20726571756573742073686f72746c6973745d2868747470733a2f2f6769746875622e636f6d2f676974636f696e636f2f7765622f6973737565732f31353530295c725c6e5c725c6e2a2a434f4d4d554e495459204d454d424552532a2a5c725c6e4665656c206672656520746f20616464207265616374696f6e732f7570766f746520796f7572206661766f72697465206e657720617661746172206974656d732f61737365747321222c226e6574776f726b223a226d61696e6e6574222c226f72675f6e616d65223a22676974636f696e636f222c22706b223a343932322c2269737375655f6465736372697074696f6e5f74657874223a2223232320557365722053746f72795c725c6e5c725c6e4173206120757365722c20492077616e7420746f206e6577206176617461722061737365747320746f2073686f77206f666620616e642065787072657373206d7973656c66206469676974616c6c792e5c725c6e5c725c6e232323204465736372697074696f6e5c725c6e5c725c6e2a547970652a3a20204e657720696d706c656d656e746174696f6e5c725c6e5c725c6e23232320446566696e6974696f6e206f6620446f6e655c725c6e5c725c6e2d205b205d2043726561746520616e202e73766720617373657420616e6420706c61636520697420696e20606170702f6173736574732f76322f696d616765732f6176617461722f605c725c6e2d205b205d204164642074686520535647206173736574206173206120602e7478746020446a616e676f2074656d706c61746520696e20606170702f6176617461722f74656d706c617465732f2f2e747874602077697468207661726961626c652066696c6c2e5c725c6e2d205b205d2050726f7669646520612073637265656e73686f74206f6620746865206e65772061737365742066726f6d2074686520617661746172206275696c64657220414e44207468652070726f66696c652064657461696c732070616765206f6e2074686973204769746875622069737375652e5c725c6e5c725c6e232323204164646974696f6e616c20496e666f726d6174696f6e5c725c6e5c725c6e4665656c206672656520746f207375626d697420616e792061737365747320796f7520776f756c64206c696b6520746f20736565206f6e20476974636f696e2e20204966207468652061737365742069732061636365707465642c20796f75722061737365742077696c6c206265636f6d6520612070617274206f6620746865206c69766520617661746172206275696c646572212020416c6c206f74686572206173736574732077696c6c20626520657363616c6174656420746f2074686520636f7265207465616d20666f722072657669657720616e6420706f74656e7469616c206d6f64696669636174696f6e2e5c725c6e5c725c6e53656520746865205b617661746172206974656d20726571756573742073686f72746c6973745d2868747470733a2f2f6769746875622e636f6d2f676974636f696e636f2f7765622f6973737565732f31353530295c725c6e5c725c6e2a2a434f4d4d554e495459204d454d424552532a2a5c725c6e4665656c206672656520746f20616464207265616374696f6e732f7570766f746520796f7572206661766f72697465206e657720617661746172206974656d732f61737365747321222c227374616e646172645f626f756e746965735f6964223a3937352c22776562335f74797065223a22626f756e746965735f6e6574776f726b222c2263616e5f7375626d69745f61667465725f65787069726174696f6e5f64617465223a747275652c226769746875625f69737375655f6e756d626572223a313636332c226769746875625f6f72675f6e616d65223a22676974636f696e636f222c226769746875625f7265706f5f6e616d65223a22776562222c226964785f737461747573223a226f70656e222c22746f6b656e5f76616c75655f74696d655f706567223a22323031382d30382d32305430363a34353a35382e3437343439355a222c2266756c66696c6c6d656e745f61636365707465645f6f6e223a6e756c6c2c2266756c66696c6c6d656e745f7375626d69747465645f6f6e223a6e756c6c2c2266756c66696c6c6d656e745f737461727465645f6f6e223a22323031382d30382d31385430303a34393a35362e3135383434335a222c2263616e63656c65645f6f6e223a6e756c6c2c22616374696f6e5f75726c73223a7b2266756c66696c6c223a222f69737375652f66756c66696c6c3f706b3d34393232266e6574776f726b3d6d61696e6e6574222c22696e637265617365223a222f69737375652f696e6372656173653f706b3d34393232266e6574776f726b3d6d61696e6e6574222c22616363657074223a222f69737375652f6163636570743f706b3d34393232266e6574776f726b3d6d61696e6e6574222c2263616e63656c223a222f69737375652f63616e63656c3f706b3d34393232266e6574776f726b3d6d61696e6e6574222c227061796f7574223a222f69737375652f7061796f75743f706b3d34393232266e6574776f726b3d6d61696e6e6574222c22636f6e74726962757465223a222f69737375652f636f6e747269627574653f706b3d34393232266e6574776f726b3d6d61696e6e6574222c22616476616e6365645f7061796f7574223a222f69737375652f616476616e6365645f7061796f75743f706b3d34393232266e6574776f726b3d6d61696e6e6574222c22736f6369616c5f636f6e747269627574696f6e223a222f69737375652f736f6369616c5f636f6e747269627574696f6e3f706b3d34393232266e6574776f726b3d6d61696e6e6574227d2c2270726f6a6563745f74797065223a22636f6f7065726174697665222c227065726d697373696f6e5f74797065223a227065726d697373696f6e6c657373222c2261747461636865645f6a6f625f6465736372697074696f6e223a22222c226e656564735f726576696577223a66616c73652c226769746875625f69737375655f7374617465223a226f70656e222c2269735f69737375655f636c6f736564223a66616c73652c226164646974696f6e616c5f66756e64696e675f73756d6d617279223a7b22746f6b656e73223a7b7d2c227573645f76616c7565223a307d7d). is it to transfer data from view to view?  if so, why not just add `?pk=<pk>` to the view and load the appro info from the database?
3. i dont think we want to let the user edit their issue desc / title on this view... i'd rather point them to github.
4. submit button on this page doesnt seem to do anything.. nor does their seem to be any validation on the page..


happ y to QA more when the items in the definition of done on the spec are completed

File: app/assets/v2/js/pages/bounty_details.js [hunk 1]
```diff
@@ -754,6 +754,8 @@ var do_actions = function(result) {
   const show_suspend_auto_approval = document.isStaff && result['permission_type'] == 'approval';
   const show_admin_methods = document.isStaff;
   const show_moderator_methods = document.isModerator;
+  const show_change_bounty = isBountyOwner(result);
```
Comment (owocki) [inline app/assets/v2/js/pages/bounty_details.js]: or is_admin

File: app/dashboard/templates/bounty/change.html [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+<html lang="en">
+
+<head>
+  {% include 'shared/head.html' %}
+  {% include 'shared/cards.html' %}
+  <link rel="stylesheet" href={% static "v2/css/submit_bounty.css" %}>
+</head>
+
+<body class="interior submit_bounty" style="background-color: #FFFFFF;">
+  {% include 'shared/tag_manager_2.html' %}
```
Comment (owocki) [inline app/dashboard/templates/bounty/change.html]: why build a whole new form. why not just re-use the `new_bounty` form?

File: app/dashboard/views.py [hunk 3]
```diff
@@ -1444,3 +1446,95 @@ def new_bounty(request):
...
+            }, status=401)
+
+        item = int(params.get('expirationTimeDelta', 0))
+        if item:
+            deadline = round(time.time() + item)
+            bounty.expires_date = timezone.make_aware(
+                timezone.datetime.fromtimestamp(deadline),
+                timezone=UTC)
+
+        item = params.get('attached_job_description', '')
```
Comment (owocki) [inline app/dashboard/views.py]: i wonder if we could use django forms for any of this repetitive code.... cc @mbeacom

File: app/retail/templates/emails/bounty_changed.html [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+{% endcomment %}
+{% load i18n humanize %}
+
+{% block content %}
+
+<h1>{% trans "Bounty Details Changed" %}</h1>
+<br>
+
```
Comment (owocki) [inline app/retail/templates/emails/bounty_changed.html]: should we list out the details that have changed.. ?

File: app/retail/templates/emails/bounty_changed.txt [hunk 5]
```diff
@@ -0,0 +1,3 @@
+Bounty Details Changed
+
```
Comment (owocki) [inline app/retail/templates/emails/bounty_changed.txt]: no data in this email

**Comment by pinkiebell** [comment]
@owocki 

Thanks for review 👍 

> 1. doesnt seem like the user can edit any of the details around project type
That's odd, seems to be a conflict with new changes on master since I worked on it.
Will fix.

> 2. why is the URL for 'change details' so long? example URL. is it to transfer data from view to view? if so, why not just add ?pk=<pk> to the view and load the appro info from the database?
Yep, I transfer the bounty details with this method.
 I will change it to use the pk= query parameter.
 
> 3. i dont think we want to let the user edit their issue desc / title on this view... i'd rather point them to github.
That I wasn't sure about. Will be removed.

> 4. submit button on this page doesnt seem to do anything.. nor does their seem to be any validation on the page..
The same problem like 1.

File: app/dashboard/templates/bounty/change.html (hunk 2 above)
Comment (pinkiebell) [reply app/dashboard/templates/bounty/change.html]: I take a look at the new changes and modify the template if necessary for easier reuse. 👍

**Comment by pinkiebell** [comment]
@owocki

Ready for review (again) 👍 .
The only open question is the additional content of the email templates; if wanted.

**Comment by thelostone-mc** [comment]
@pinkiebell quick rebase and then we merge ?

**Comment by pinkiebell** [comment]
@thelostone-mc Done! 👍

File: app/retail/templates/emails/bounty_changed.html (hunk 4 above)
Comment (thelostone-mc) [reply app/retail/templates/emails/bounty_changed.html]: I don' think that would be needed! cause the funder would see it on the bounty page upon redirection

**Comment by thelostone-mc** [comment]
@PixelantDesign this is how it looks
x

@pinkiebell Oh i just noticed should we update the placeholder text ? 
Along the lines to -> `Update your bounty settings to get the right crowd` ? @vs77bb might have something better :P

**Comment by pinkiebell** [comment]
@thelostone-mc 
Did you mean the button on the bounty details page or on the bounty change page?
Instead of 'Details'- at the top?

**Comment by thelostone-mc** [comment]
@pinkiebell the button on the bounty details page

**Comment by pinkiebell** [comment]
@thelostone-mc Done 👍

**Comment by thelostone-mc** [comment]
sweet will merge this in once @PixelantDesign reviews it 👍

**Comment by PixelantDesign** [comment]
My other question is, if a funder is updating the issue details..should the following also happen...
1) Gitcoin Bot should **update** the thread with the revised details.
2) The **funder** should receive a confirmation email of the change.
3) **Anyone** that has **Started Work** on the issue should also receive an email confirmation.

Thoughts from the group? @owocki @vs77bb @thelostone-mc @pinkiebell.

**Comment by pinkiebell** [comment]
@PixelantDesign 
Copy update done!
Regarding your questions, 2. and 3. are already the case with this PR.
For number 1. any ideas how that should look like?

**Comment by vs77bb** [comment]
LGTM! 

> Gitcoin Bot should update the thread with the revised details.

I think we should hold off on this for now. There's already enough comments on the Github thread, IMO. If it turns out to be an issue, we could include 'Approval' vs. 'Permissions' bounties in a v2 of the PR @oogetyboogety is working on regarding the Gitcoin Bot copy updates!

**Comment by PixelantDesign** [comment]
sounds good. Let's merge @thelostone-mc @SaptakS ?

**Comment by thelostone-mc** [comment]
@pinkiebell thanks for the contribution ^_^

**Comment by vs77bb** [comment]
Sweeeeeeeeeetttt!

**Pull request merged**